vpx_ports/ppc_cpudetect.c

```diff
@@ -0,0 +1,36 @@
+#if defined(__linux__)
+#include <fcntl.h>
+#include <unistd.h>
+#include <asm/cputable.h>
+#include <linux/auxvec.h>
+#endif
+
+int ppc_simd_caps(void) {
+	int result = 0;
+#if defined(__linux__)
```

Comment:
Seems matching what happens in other architectures, thank you!

You should also consider `CONFIG_RUNTIME_CPU_DETECT` from what I'm seeing.